packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -323,6 +323,27 @@ new LayerVersion(this, 'MyLayer', {
 });
 ```
 
+## Lambda Insights
```

Comment:
How about just providing these as a set of `ILayerVersion` implementations instead?

The user experience will then look like - 

```ts
const lambda = new Function(...);
lambda.addLayers(InsightsLayerVersion.VERSION_1_0_98_0);
```

Feels simpler.

Reply:
I wanted to do that originally, but the ARN isn't the same for each region, so I can't make an assumption about the ARN just based on the provided version.
https://github.com/aws/aws-cdk/pull/15439/files#diff-203458110222061ed1f54590e8556cf107bfa77c6da383e05f955491045e33faR96-R118